website/content/docs/upgrading/upgrade-to-1.18.x.mdx

```diff
@@ -0,0 +1,20 @@
+---
+layout: docs
+page_title: Upgrade to Vault 1.18.x - Guides
+description: |-
+  Deprecations, important or breaking changes, and remediation recommendations
+  for anyone upgrading to 1.18.x from Vault 1.18.x.
+---
+
+# Overview
+
+The Vault 1.18.x upgrade guide contains information on deprecations, important
+or breaking changes, and remediation recommendations for anyone upgrading from
+Vault 1.17. **Please read carefully**.
+
+## Important changes
+
+### Activity Log Changes
+
+The field `default_report_months` can no longer be configured. Attempts to write
+a value to the field will not result in any change.
```

Comment:
Slight tweak to the suggestion made above:

````suggestion
The `default_report_months` field is now read-only.

If you attempt to modify `default_report_months` through the
[/sys/internal/counters/config](/vault/api-docs/system/internal-counters#update-the-client-count-configuration)
endpoint, Vault ignores the update and returns a warning:

<CodeBlockConfig hideClipboard>

```shell-session
<code-quoted example>
```

</CodeBlockConfig>

````